python/services/dataservice/README.md

```diff
@@ -2,4 +2,74 @@
 Python package for data management service for DMOD infrastructure.
 
 # Structure
-Structure has been modified from original to have inner duplicate directory in order to comply with general Python packaging structure.  This facilitates executing tests in a variety of different scenarios, including running integration tests on a local machine using the included test script and/or running unit tests directly within an IDE.
+Structure has been modified from original to have inner duplicate directory in order to comply with general Python packaging structure.
+This facilitates executing tests in a variety of different scenarios, including running integration tests on a local machine using the included test script and/or running unit tests directly within an IDE.
+
+## Configuration
+
+Provide listed service configuration variables as either environment variables, in a '.env' file, as a `secret` in `/run/secrets`, or as a mixture of the aforementioned options.
+
+Configuration option names are case insensitive.
```

Comment:
I think this would be helpful, but not necessary:

```suggestion
Configuration variable names are case insensitive.
```

Reply:
Yeah, I went back and forth when writing this. We usually say a CLI tools command line options, not its command line variables. I think it depends on your background which term you prefer. I am going to leave it as is. Thanks for the careful read and feedback!